tests/AvroConvertTests/Headless/HeadlessComponentTests.cs

```diff
@@ -30,5 +30,23 @@ public void Component_SerializeHeadlessBiggerObjectAndReadSmaller_NoError()
             Assert.NotNull(deserialized);
             Assert.Equal(toSerialize, deserialized);
         }
+
+        [Fact]
+        public void Component_DeserializeWithMissingFields_NoError()
+        {
+            //Arrange
+            SimpleClass toSerialize = _fixture.Create<SimpleClass>();
```

Comment:
please use in the test BaseTestClass and ReducedBaseTestClass - do not create new.